modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -339,30 +334,30 @@ Euclidean similarity is useful when the _distance_ between the vectors is what d
 A valid vector for a Euclidean vector index is when all vector components can be represented finitely in {ieee-754} single precision.
 
 Euclidean interprets the vectors in Cartesian coordinates.
-The measure is related to the Euclidean distance, how far two points are from one another.
-However, that is unbounded and less useful as a similarity score.
-Euclidean similarity bounds the square of Euclidean distance.
+The measure is related to the Euclidean distance, i.e. how far two points are from one another.
```

Comment:
I'm aware that `i.e.` _without_ the trailing comma is the British English style, and _with_ the trailing comma (`i.e.,`) is the American English style.

I was under the impression we (unfortunately) use the American conventions; therefore, should we have the trailing comma, or use some English connector instead?

Reply:
I will add the comma (and strong agree on the preferred use of convention)